File: lanjii/lanjii-modules/module-system/system-biz/src/main/java/com/lanjii/sys/entity/SysConfig.java
package com.lanjii.sys.entity;

import com.baomidou.mybatisplus.annotation.TableName;
import com.lanjii.sys.base.AdminBaseMapper;
import com.lanjii.system.api.dto.SysConfigDTO;
import com.lanjii.system.api.vo.SysConfigVO;
import com.lanjii.framework.mp.base.BaseEntity;
import lombok.Data;
import lombok.EqualsAndHashCode;
import org.mapstruct.Mapper;
import org.mapstruct.factory.Mappers;

/**
 * 系统配置表(SysConfig)表实体类
 *
 * @author lanjii
 */
@Data
@EqualsAndHashCode(callSuper = true)
@TableName("sys_config")
public class SysConfig extends BaseEntity {

    /**
     * 配置ID
     */
    private Long id;

    /**
     * 配置名称
     */
    private String configName;

    /**
     * 配置键名
     */
    private String configKey;

    /**
     * 配置键值
     */
    private String configValue;

    /**
     * 配置类型（1-系统配置 2-业务配置）
     */
    private Integer configType;

    /**
     * 是否启用（1启用 0禁用）
     */
    private Integer isEnabled;

    /**
     * 备注
     */
    private String remark;

    @Mapper
    public interface SysConfigMapper extends AdminBaseMapper<SysConfig, SysConfigVO, SysConfigDTO> {

    }

    public static final SysConfig.SysConfigMapper INSTANCE = Mappers.getMapper(SysConfig.SysConfigMapper.class);

}

File: lanjii/lanjii-admin-ui/src/types/sys/sysConfig.ts
/**
 * 系统配置
 */
export interface SysConfig {
  /** 配置ID */
  id?: number
  /** 配置名称 */
  configName: string
  /** 配置键名 */
  configKey: string
  /** 配置键值 */
  configValue: string
  /** 配置类型（1系统/2业务） */
  configType?: number
  /** 是否启用（1启用/0禁用） */
  isEnabled?: number
  /** 备注 */
  remark?: string
  /** 创建时间 */
  createTime?: string
  /** 更新时间 */
  updateTime?: string
  /** 创建人 */
  createBy?: string
  /** 更新人 */
  updateBy?: string
  /** 删除标志（0-未删除，1-已删除） */
  deleted?: number
}
